lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -1011,22 +1011,20 @@ export async function updatePr({
       };
     }
 
-    const { body: updatedPr } = await bitbucketServerHttp.putJson<{
-      version: number;
-      state: string;
-    }>(
+    const { body: updatedPr } = await bitbucketServerHttp.putJson<BbsRestPr>(
       `./rest/api/1.0/projects/${config.projectKey}/repos/${config.repositorySlug}/pull-requests/${prNo}`,
       { body },
     );
 
-    updatePrVersion(prNo, updatedPr.version);
+    updatePrVersion(prNo, updatedPr.version!);
 
     const currentState = updatedPr.state;
     // TODO #22198
     const newState = {
       ['open']: 'OPEN',
       ['closed']: 'DECLINED',
     }[state!];
+    let finalState: string = currentState;
```

Comment:
why do we need this `string` cast?

Reply:
Nice catch! Thanks... It was a wrong cast and wrong value too.